Source/UnrealExamples/Public/WebMessage.h

```diff
@@ -0,0 +1,21 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+
+class UNREALEXAMPLES_API WebMessage
+{
+public:
+	FString Type;
+	FString Source;
```

Comment:
is this field used?

Reply:
Right source is only used javascript side, probably don't need to have it available to devs as it is currently always the same. (at least it should be) not sure if there are plans to change that